certmanager/generate_falcon_chains.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash 
+
+# Script to generate Falcon 512 and Falcon1024 certificate chains.
+#
+# Copyright 2021 wolfSSL Inc. All rights reserved.
+# Original Author: Anthony Hu.
+# Execute in openssl directory after building oqs fork of OpenSSL. 
+
+# Generate conf files.
+printf "\
+[ req ]\n\
+prompt                 = no\n\
+distinguished_name     = req_distinguished_name\n\
+\n\
+[ req_distinguished_name ]\n\
+C                      = CA\n\
+ST                     = ON\n\
+L                      = Waterloo\n\
+O                      = wolfSSL Inc.\n\
+OU                     = Engineering\n\
+CN                     = Root Certificate\n\
+emailAddress           = root@wolfssl.com\n\
+\n\
+[ ca_extensions ]\n\
+subjectKeyIdentifier   = hash\n\
+authorityKeyIdentifier = keyid:always,issuer:always\n\
+keyUsage               = critical, keyCertSign\n\
+basicConstraints       = critical, CA:true\n" > root.conf
+
+printf "\
+[ req ]\n\
+prompt                 = no\n\
+distinguished_name     = req_distinguished_name\n\
+\n\
+[ req_distinguished_name ]\n\
+C                      = CA\n\
+ST                     = ON\n\
+L                      = Waterloo\n\
+O                      = wolfSSL Inc.\n\
+OU                     = Engineering\n\
+CN                     = Entity Certificate\n\
+emailAddress           = entity@wolfssl.com\n\
+\n\
+[ x509v3_extensions ]\n\
+subjectKeyIdentifier   = hash\n\
+authorityKeyIdentifier = keyid:always,issuer:always\n\
+keyUsage               = critical, digitalSignature\n\
+extendedKeyUsage       = critical, serverAuth,clientAuth\n" > entity.conf
+
+###############################################################################
+# Falcon 512
+###############################################################################
+
+# Generate root key and entity private keys. 
+./apps/openssl genpkey -algorithm falcon512 -outform pem -out falcon512_root_key.pem
```

Comment:
Can you make the `openssl` an environment variable? Most people don't have `./apps/openssl`.

Reply:
I think it would be best to add a reference to wolfSSL/INSTALL file saying there are instructions on how to download and build OQS OpenSSL.

Reply:
But will also make env variable.